lib/public_key/asn1/CMSAesRsaesOaep.asn1

```diff
@@ -1,3 +1,40 @@
+-- %CopyrightBegin%
+--
+-- SPDX-License-Identifier: NOASSERTION
```

Comment:
I believe this should reference the IETF license:
https://scancode-licensedb.aboutcode.org/ietf.html

```suggestion
-- SPDX-License-Identifier: LicenseRef-scancode-ietf
```